### ISSUE TITLE
large contents in td (larger than a page) make a mess in pagination

### ISSUE THREAD

**Issue by GoogleCodeExporter** [jkoo7yeha/dompdf#310]
```
What steps will reproduce the problem?
1.
intert a table
2.
fill the td with a large content (more than 1 page)


What is the expected output? What do you see instead?

The expected output is a table cell splitted into more pages.
an number of blank pages are generated and the td content fills one page and is 
truncated at the end of the page


What version of the product are you using? On what operating system?

version 0.6.0 beta2 on Debian/GNU Squeeze with PHP 5.3.3 and Apache 2.2.16


Please provide any additional information below.

```

Original issue reported on code.google.com by `gianluca...@gmail.com` on 23 Jun 2011 at 3:13
- Merged into: #91

**Comment by GoogleCodeExporter** [comment]
```
While not technically a duplicate of issue 91, what you are seeing is a result 
of a temporary hack around the exhaustion of resources caused by that issue. We 
expect to have a fix ready for the final 0.6.0 release.
```

Original comment by `eclecticgeek` on 23 Jun 2011 at 9:01
- Changed state: **Duplicate**

**Comment by GoogleCodeExporter** [comment]
Original comment by `eclecticgeek` on 30 May 2013 at 5:16
- Added labels: **Restrict-AddIssueComment-Commit**
